assets/sass/vendor/_mdc-radio.scss

```diff
@@ -30,4 +30,12 @@
 		width: 100%;
 		z-index: 1;
 	}
+
+	// The Radio component has an essential padding of 11 pixels to accommodate
+	// its hover/focus background. This makes it not possible to align the radio
+	// to the absolute left when required. The following style compensates for the
+	// padding and aligns it to the absolute left.
+	&.mdc-radio--align-left {
+		margin-left: -10px;
```

Comment:
The comment here says 11px but this is only 10. Also, the file `assets/sass/components/user-input/googlesitekit-user-input-preview.scss` uses 10px for both checkbox and radios. It's fine if they should be different for accuracy, but just noting this difference from the previous CSS.

Is there a custom property that material uses for the padding that we should reference here instead?

Reply:
> The comment here says 11px but this is only 10. Also, the file `assets/sass/components/user-input/googlesitekit-user-input-preview.scss` uses 10px for both checkbox and radios. It's fine if they should be different for accuracy, but just noting this difference from the previous CSS.

Ah, I copy pasted the comment from Checkbox but mistakenly didn't update this detail, updated it now, thank you!

> Is there a custom property that material uses for the padding that we should reference here instead?

[According to the material codebase](https://github.com/material-components/material-components-web/blob/b994146f6d849bd80f280c147d64adae5b930a3c/packages/mdc-checkbox/_checkbox-theme.scss#L342), it seems to be the ripple size minus icon size divided by 2. Do you think there is something usable here?

Reply:
Hmm that looks complicated 😄 There's probably a way to do it properly with a Material theme mixin but it's not worth spending that much time on.

Reply:
I agree 😄 Thank you!